### PR TITLE
Modularized main.py and added aws account migration on pulumi destroy process

### PR DESCRIPTION
**Pull request by shawngerrard**
I've modularized `main.py` through the creation of `users.py`, `org.py`, `ou.py`, `account.py`, and `policies.py`.

The modularization wasn't a straight copy-paste so some of the references may need to be updated in these files as well as `Pulumi.dev.yaml`, as I'm quite sure there's better ways to obtain values etc., however this is a fairly decent first cut of the end-to-end pulumi stack, if I do say so myself. So in saying that, I expect some further tweaks to do here.

This update assumes the aws account has been created and if not will attempt to create it. However, I haven't tested the account creation scenario yet. Otherwise, the stack comes up and destroys successfully, moving the account appropriately before proceeding with the actions.

When the PR is merged, I'll close the related issues.

Any questions, sing out!

### CODE REVIEW

File: org-asterion/config/Pulumi.dev.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+config:
+  aws:defaultTags:
+    tags:
+      environment: dev
+  aws:region: ap-southeast-2
+  org-asterion:accountId: 822258672282
+  org-asterion:currentOrgName: shawng
```
Comment (jmhbnz) [inline org-asterion/config/Pulumi.dev.yaml]: Are these values valid for asterion? The look like personal account.

File: org-asterion/config/Pulumi.dev.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  org-asterion:environments: 
+    - dev
+    - test
+    - prod
+  org-asterion:iamGroupName: asterion-admins
+  org-asterion:iamUsersToAdd: 
+    users:
+    - shawn
+    - james
+    - dj
```
Comment (jmhbnz) [inline org-asterion/config/Pulumi.dev.yaml]: Can we please use full first names to match our emails, i.e. `daljit`.

**Review by jmhbnz — CHANGES_REQUESTED** [review]

**Comment by jmhbnz** [comment]
One other suggestion would be that we display the diagram svg in the `README.org` for `org-asterion`.

File: org-asterion/config/Pulumi.dev.yaml (hunk 1 above)
Comment (shawngerrard) [reply org-asterion/config/Pulumi.dev.yaml]: `currentOrgName` actually relates to the pulumi org, not the aws asterion account. This was required for any stack references, which are now no longer part of the solution so I've removed all references of this.

File: org-asterion-dev/Pulumi.dev.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+config:
```
Comment (jmhbnz) [inline org-asterion-dev/Pulumi.dev.yaml]: Is this file supposed to be here?

**Review by jmhbnz — APPROVED** [review]
Approved - just one file to tidy up!

File: org-asterion-dev/Pulumi.dev.yaml (hunk 3 above)
Comment (shawngerrard) [reply org-asterion-dev/Pulumi.dev.yaml]: I had to reverse a few commits so looks like a straggler got in there - removed it now.

**Pull request merged**